src/test/tests/unit/annotation_objects.py

```diff
@@ -0,0 +1,32 @@
+# ----------------------------------------------------------------------------
+#  CLASSES: nightly
+#
+#  Test Case:  annotation_objects.py
+#
+#  Tests:      AnnotationObject unit test
+#
+#  Alister Maguire, Fri Jul 16 12:50:11 PDT 2021
+#
+#  Modifications:
+#
+# ----------------------------------------------------------------------------
+
+
+def CheckComparisonOperator():
+    annot = CreateAnnotationObject("Text2D")
+
+    #
+    # In the past, a bug caused a crash whenever we compared to a non-annotation
+    # type object.
+    #
+    result = ""
+    result += str(annot == "") + "\n"
+    result += str(annot == annot)
+    
+    TestText("ComparisonOperator_00", result)
```

Comment:
Instead of TestText, which needs a baseline, why not change this to use TestValueEQ?
I believe the syntax would be:
TestValueEQ(annot == "", False)
TestValueEQ(annot==annot, True)

Reply:
Ah, that's a great idea! I'll go ahead and make that change.